test/Grpc.IntegrationTests/OrchestrationErrorHandling.cs

```diff
@@ -136,7 +136,7 @@ public async Task RetryActivityFailures(int expectedNumberOfAttempts)
                 {
                     await ctx.CallActivityAsync("Foo", options: retryOptions);
                 })
-                .AddActivity("Foo", context =>
+                .AddActivity("Foo", (TaskActivityContext context) =>
```

Comment:
Curious: was this change caused by an overload ambiguity introduced in this PR?

Reply:
Yes, exactly that. There is ambiguity between `Func<IServiceProvider, ITaskActivity>` and `Action<TaskActivityContext>` - this is because it throws an exception and return value cannot be inferred. Only a minor inconvenience because I imagine customers will typically have a return value (which will help resolve the correct overload).